compose/common.yml

```diff
@@ -307,6 +332,19 @@ services:
     ports:
       - ${API_PORT:-4000}:3000
 
+  provider-server-blockfrost:
+    <<:
+      - *from-sdk
+      - *logging
+      - *provider-server-blockfrost
+      - *with-postgres
+    environment:
+      <<:
+        - *sdk-environment
+        - *provider-server-environment-blockfrost
+    ports:
+      - ${API_PORT:-4000}:3000
```

Comment:
Doesn't this conflicts with `provider-server`?

Reply:
Docker-related changes are no longer a part of this PR.